devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -223,3 +237,54 @@ def setLevel(level) {
 		zwave.switchMultilevelV1.switchMultilevelGet().format()
 	], 5000)
 }
+
+def configure() {
+	log.debug "Configure..."
+    def result = []
+
+    zwave.manufacturerSpecificV2.manufacturerSpecificGet().format()
+
+    log.debug "Configure zwaveInfo: "+zwaveInfo
+
+    if (zwaveInfo.mfr == "0086") {  // Aeon Labs meter
+		result << response(delayBetween([
+            zwave.configurationV1.configurationSet(parameterNumber: 80, size: 1, scaledConfigurationValue: 2).format(),    // basic report cc
+			zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 4).format(),   // report power in watts
+			zwave.configurationV1.configurationSet(parameterNumber: 111, size: 4, scaledConfigurationValue: 300).format(), // every 5 min
+            zwave.configurationV1.configurationSet(parameterNumber: 102, size: 4, scaledConfigurationValue: 8).format(),   // report energy in kWh
+			zwave.configurationV1.configurationSet(parameterNumber: 112, size: 4, scaledConfigurationValue: 300).format(), // every 5 min
+            //zwave.configurationV1.configurationSet(parameterNumber: 103, size: 4, scaledConfigurationValue: 0).format(),    // no third report
+			//zwave.configurationV1.configurationSet(parameterNumber: 113, size: 4, scaledConfigurationValue: 300).format(), // every 5 min
+			meterGet(scale: 0).format(),
+			meterGet(scale: 2).format(),
+		]))
+	} else {
+		result << response(delayBetween([
+			meterGet(scale: 0).format(),
+			meterGet(scale: 2).format(),
+		]))
+	}
+    result
+}
+
+def reset() {
+	return [
+		meterReset().format(),
+		meterGet(scale: 0).format()
+	]
+}
+
+def meterGet(scale)
+{
+    return (isNano() ? zwave.meterV3.meterGet(scale) : zwave.meterV2.meterGet(scale))
```

Comment:
Meter V3 GET and Meter V2 GET are identical. You don't need to separate them like this. Also you need to make sure that you are passing a map of parameters to the `meterGet` method. I think right now you are just passing an integer?